mod-login-saml/src/main/resources/domain/mod-login-saml/features/loginSamlRedirect.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Login SAML with a REDIRECT binding
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers =
+    """
+    {
+      "X-Okapi-Tenant": "#(testTenant)",
+      "X-Okapi-Token": "#(okapitoken)",
+      "Accept": "application/json, text/plain"
+    }
+    """
+
+  Scenario: Do preflight request, POST to saml login endpoint, and receive correct response for REDIRECT binding
+    Given path "saml/login"
+    And header Origin = baseUrl
+    And header Access-Control-Request-Method = "POST"
+    When method OPTIONS
+    Then status 204
+    And match header access-control-allow-methods contains "POST"
+    And match header access-control-allow-origin == "*"
+    Given path "saml/login"
```

Comment:
Can you add an empty line before `Given` for readability?

Reply:
Yes I can add that. Empty lines in scenarios don't appear to cause problems and it would make it easier to follow.